www/src/pages/components/card.mdx

```diff
@@ -87,4 +150,48 @@ import SingleComponentStatus from '../../components/SingleComponentStatus';
     </Card.Footer>
   </Card>
 </CardDeck>
-```
\ No newline at end of file
+```
+
+##### Props
+
+##### CardGrid
+
+<StaticQuery
+  query={graphql`
+    query {
+      componentMetadata(displayName: { eq: "CardGrid" }) {
+        ...ComponentDocGenData
+      }
+    }
+  `}
+  render={({ componentMetadata }) => {
+    if (componentMetadata == null) return null;
+    return <PropsTable propMetaData={componentMetadata.props} />;
+  }}
+/>
+
+##### Theme variables (SCSS)
```

Comment:
What is the convention we have for these theme variables in the docs? Do we plan to essentially copy/paste the entire _variables.scss file for each component into the docs? If so, I wonder if it might make sense to figure out how to automate that across the repo similar to the auto-documenting props 🤔

Reply:
Yes and yes.  If we stick with sass or css variables for theming then yes we should automate this.

Reply:
Cool, I backlogged a Paragon ticket to automate this if we end up wanting to go that route: https://openedx.atlassian.net/browse/PAR-364